jobs/start-regulering/README.md

```diff
@@ -5,12 +5,11 @@ ___
 
 For å trigge denne jobben gjøres følgende:
 1. Finn docker image versjon som skal benyttes av denne jobben.
-   1. Gå til siste bygg i [GitHub actions](https://github.com/navikt/pensjon-etterlatte/actions/workflows/job-post-til-kafka.yaml) og sjekk loggene for image versjon.
+   1. Gå til siste bygg i [GitHub actions](https://github.com/navikt/pensjon-etterlatte-saksbehandling/pkgs/container/pensjon-etterlatte-saksbehandling%2Fstart-regulering) og sjekk loggene for image versjon.
```

Comment:
Eg gjorde noko lurt på barnetrygd for å skrive ut image-versjonen i byggejobben. Noterer på blokka å finne fram den, det var generelt hendig.

Reply:
ooh, hade vært kult det! 🕶️